### ISSUE TITLE
Bug in storage.select(date("now", "start of month", "+1 month", "-1 day")).front()

### ISSUE THREAD

**Issue by juandent** [fnc12/sqlite_orm#181]
it ignores -1 day. Also buggy is:

```
auto nowWithOffset2 = storage.select(date("now", "+2 day")).front();
cout << "SELECT date('now','+2 day') = " << nowWithOffset2 << endl;
```
I ran both of this in DB Browser for SQLite and ran perfectly...

Is this too hard to fix?

Thanks,
Juan

**Comment by fnc12** [comment]
`storage.select(date("now", "start of month", "+1 month", "-1 day"))` generates `SELECT DATE('now', '-1 day', '+1 month', 'start of month')`. Looks like it is bug with the order. Thanks

**Comment by fnc12** [comment]
fixed in #182 . Available in `dev` branch. Also will be merged in the next release
